_dashboards/visbuilder.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: VisBuilder
+nav_order: 8
+---
+
+# VisBuilder
+
+VisBuilder is an experimental feature and shouldn't be used in a production environment. For updates on its progress, or if you want to leave feedback that helps improve the feature, see the [git issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2280).
+{: .warning}
+
+You can use the VisBuilder visualization type in OpenSearch Dashboards to create data visualizations easily using a drag-and-drop gesture. With VisBuilder you have:
```

Comment:
Be cautious when suggesting to a user that something will be “easy” or “is easily done.” Our users vary in terms of sophistication, skill level, and familiarity with OpenSearch, so ease of use will differ across our user base.

Reply:
Got it :)